### PR TITLE
[4.12] Adding QE profiles for ARM AWS IPI

### PR DESCRIPTION
**Pull request by aleskandro**
/hold

### CODE REVIEW

**Comment by aleskandro** [comment]
Refers ARMOCP-446

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__arm64-nightly.yaml [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: m6g.large
+      E2E_RUN_TAGS: '@aws-ipi and @arm64 and @network-openshiftsdn and not @fips'
+      OCP_ARCH: arm64
+      TAG_VERSION: '@4.12'
+    test:
+    - chain: openshift-e2e-test-qe
+    workflow: cucushift-installer-rehearse-aws-ipi
+- as: e2e-aws-ipi-ovn-ipsec
+  cluster: build05
```
[the remark this answers is not in the document]
Comment (aleskandro) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__arm64-nightly.yaml:63]: no, there is not... i had to manually test another thing on that side and missed to remove it later. Removing

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__arm64-nightly.yaml [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+      version: "4.12"
+resources:
+  '*':
+    limits:
+      memory: 8Gi
+    requests:
+      cpu: "1"
+      memory: 4Gi
+tests:
+- as: e2e-aws-ipi
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__arm64-nightly.yaml:47]: Perhaps we should follow the job naming here, https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/README.md

**Pull request merged**